pages/troubleshooting/faq.md

```diff
@@ -120,3 +121,6 @@ The resin.io agent needs to be able to access our cloud services in order for yo
 
 ##### How do you secure your own "cloud" to prevent malicious attack which may allow attacker to break-in our systems?
 Generally we try to follow good OPSEC practices for our systems. We support 2FA for user accounts and force all the connections to be over HTTPS.
+
+##### What does it mean when a device type is discontinued?
+Discontinued devices will no longer be actively supported by resin.io. This means we will no longer provide prebuilt versions of resinOS for these devices, and we will not be resolving any issues related to these boards. Please contact sales@resin.io with any questions regarding continued device support.
```

Comment:
We should also add the at we wont allow the creation of new applications for these device types, but already existing applications and the devices associated with it will continue to function as normal, but will not have new OS or supervisor releases.